util/reggen/signal.py

```diff
@@ -5,30 +5,34 @@
 from typing import Dict, Sequence
 
 from reggen.bits import Bits
-from reggen.lib import check_keys, check_name, check_str, check_int, check_list
+from reggen.lib import check_keys, check_name, check_str, check_int, check_list, check_bool
 
 
 class Signal:
 
-    def __init__(self, name: str, desc: str, bits: Bits):
+    def __init__(self, name: str, desc: str, bits: Bits, enabled_after_reset: bool = False):
```

Comment:
I'm not really familiar with reggen, but this sounds like a change that will apply to all fields of every register. In this case wouldn't a "reset value" make more sense?